ClemBot.Bot/bot/cogs/trivia_cog.py

```diff
@@ -36,26 +36,24 @@ def __init__(self, bot):
     @ext.example("trivia")
     async def trivia(self, ctx):
 
-        async with await self.session.get(DEFAULT_URL) as resp:
-            parse_text = await resp.text()
-            new_response = json.loads(parse_text)
-            parsed_response = self.html_parser(new_response)
+        async with self.session.get(DEFAULT_URL) as resp:
+            parsed_response = self.html_parser(await resp.json())
 
-            #If you're curious as to why this doesn't check response code: It's because it will never NOT have questions for the default. If it does the website is down and it will error out anyway.
+            # If you're curious as to why this doesn't check response code: It's because it will never NOT have questions for the default. If it does the website is down and it will error out anyway.
 
         best_list = await self.dict_publisher(parsed_response)  #Publishes the dictionary
 
         new_task = await self.asyncio_publisher(ctx, best_list[1])  #This returns key values for our list
 
-          #Starts the event listener for the reaction BEFORE emojis are sent
+        # Starts the event listener for the reaction BEFORE emojis are sent
 
-        task1 = asyncio.create_task(self.send_scroll_reactions(ctx, new_task[0], new_task[1], new_task[2], new_task[4]))  #Sends the emojis for the reaction
+        task_1 = asyncio.create_task(self.send_scroll_reactions(ctx, new_task[0], new_task[1], new_task[2], new_task[4]))  #Sends the emojis for the reaction
```

Comment:
Forgor to add space on this in-line comment.
```suggestion
        task_1 = asyncio.create_task(self.send_scroll_reactions(ctx, new_task[0], new_task[1], new_task[2], new_task[4]))  # Sends the emojis for the reaction
```